scalafix/input/src/main/scala/fix/Playground.scala

```diff
@@ -0,0 +1,8 @@
+/*
+rule = "scala:fix.CrossCompat"
+ */
+package fix
+
+object Playground {
+
+}
```

Comment:
I guess this is an oversight?

Reply:
Ditto, it's useful.

Reply:
What is this file testing?

Reply:
There is no testOnly in scalafix-testkit. I mimic this with: https://github.com/scala/scala-collection-compat/blob/master/scalafix/tests/src/test/scala/fix/ScalafixTests.scala#L20